democracy/models/section.py

```diff
@@ -49,6 +49,8 @@ class Section(Commentable, StringIdBaseModel):
     content = models.TextField(verbose_name=_('content'), blank=True)
     plugin_identifier = models.CharField(verbose_name=_('plugin identifier'), blank=True, max_length=255)
     plugin_data = models.TextField(verbose_name=_('plugin data'), blank=True)
+    plugin_source = models.URLField(verbose_name=_('plugin source code'), blank=True)
```

Comment:
Right now the base plugin class doesn't give a damn about the format of the data; the idea was that should the data need some validation, that plugin class could do just that. (Or, maybe, stash the data in a new model or something? Anyway.)

To avoid hard-coding things, maybe turn all this into a `Plugin` model:

* `identifier` (not strictly necessary, I guess)
* `iframe_url`
* `fullscreen`

and add a foreign key to it in Section?